data_updates/Python/oecd_data.py

```diff
@@ -0,0 +1,124 @@
+import os
+import glob
+import numpy
+from numpy import NaN
+import pandas as pd
+from github import Github, InputGitTreeElement
+from django.conf import settings
+
+PURPOSE_CODE_TRENDS_URL = "https://staging-ddw.devinit.org/api/export/1241/"
+CSV_FOLDER = "oecd_csv"
+DATA_REPO = "devinit/di-website-data"
+REMOTE_BRANCH = "gates/oecd"
```

Comment:
Test the generated files on the `di-chart-boilerplate` repo (`page/gates` branch) to make sure the tables and charts still function, then change this to `main`.